README.md

```diff
@@ -1,4 +1,108 @@
+
+> :warning: This is a temporary (aka bootstrap) governance document that
+> is effective until the project is fully established.
+> A final process 
+
+## Project manifesto
+
+### Our goal
+
+Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
+
+### Scope
+
+The scope includes but not limited to:
+
+- An **open standard** that allows adopters and vendors to easily integrate with spec-compliant feature flag solutions
+- Creating a **vendor-agnostic ecosystem** supported by the CNCF that provide a seamless experience for developers, SREs, and vendors
+- Providing Unified API and SDK;
+- Developer-first, cloud-native reference implementation; 
+- Ensuring extensibility for open source and commercial offerings.
+
+### Why OpenFeature?
+
+There are many existing solutions, open source or not.
+Unfortunately they lack key features needed for wide adoption
+by companies and in the open source ecosystem:
+
+- **Many SDKs** doing the same thing, but slightly differently
+- Many companies **“rebuilding” the wheel**
+- SDKs are generally open source but have no ecosystem
+- **“Vendor” lock-in** at code-level
+- **No collaboration** across vendors. 
+- **Missing integration** with observability tools (e.g. OpenTelemetry)
+
+### Project Differentiators
+
+This project is more than just copy and paste of existing solutions
+
+* Current solutions often have **eval logic in SDKs**.
+  This may require **updating all your apps** for a new rule type.
+* Config in Feature Flagging tools that are **separate from the rest of the app** make it harder to use approaches like GitOps. 
+* Config and rule changes are **hard to integrate into observability** tools 
+* Information to decide on features already available in observability tools is **not leveraged and collected twice**
+* No **agreed on grammar/structure** for rules which could be automatically validated (e.g. with CUE)
+
+## Roadmap
+
+The project roadmap is available [here](https://github.com/orgs/openfeatureflags/projects/1).
+New initiatives for the roadmap can be suggested via marking an issue as `roadmap-proposal` and then reaching a community decision as documented in _Decision Making_.
+
+- [Initiatives on the project roadmap](https://github.com/search?q=org%3Aopenfeatureflags+label%3Aroadmap&type=issues)
+- [Pending roadmap suggestions](https://github.com/search?q=org%3Aopenfeatureflags+label%3Aroadmap-proposal&type=issues)
+
+## Decision making
+
+Decisions are made by a consensus of [Interested Parties](./interested-parties.md).
+If this consensus cannot be reached,
+the decision can be made by the plain majority of founding members.
+
+<!-- TODO: List founding members or delegate the decision to CDF TAG App Delivery or another entity -->
+
+Key discussions and decisions should happen asynchronously in communication channels like GitHub Issues, pull requests and project chats.
```

Comment:
I would discourage key discussions from happening in project chat where they are not easily searchable or archived.

Reply:
Me too. At the same time unfortunately it happens in almost all projects. I am fine with it happening in the bootstrap phase when not all channels are established. I will remove explicit reference in the text anyway

```suggestion
Key discussions and decisions should happen asynchronously in communication channels like GitHub Issues, discussions or pull requests.
```